packages/integrations/mdx/src/astro-data-utils.ts

```diff
@@ -24,9 +24,9 @@ export function rehypeApplyFrontmatterExport(pageFrontmatter: Record<string, any
 				jsToTreeNode(
 					/** @see 'vite-plugin-markdown' for layout props reference */
 					`import { jsx as layoutJsx } from 'astro/jsx-runtime';
-				import Layout from ${JSON.stringify(frontmatter.layout)};
 				
-				export default function ({ children }) {
+				export default async function ({ children }) {
```

Comment:
can you leave a comment outside of the string (like on line 25) mentioning that this needs to be an async import. On stuff like this it's also good to leave a date with the comment so that in a year when we're on Vite 8 the reader knows that it may be out of date.

Here's the format I always use:

```
// NOTE(fks) 08-22-2022: ...
```

Reply:
Done!